azure-pipelines.yml

```diff
@@ -144,10 +139,12 @@ stages:
               conda env create -f environment.yml
               conda activate pytools-develop
               cd $(System.DefaultWorkingDirectory)
-              pip install pytest-azurepipelines
-              coverage run -m pytest test/test/
-              coverage xml
-              coverage html
+              pytest \
+                 --cov facet \
```

Comment:
This needs to specify the module, in this case `pytools`

Reply:
Quite right! Correction made in #5892a93